frontend/src/hooks/api/relayNumber.ts

```diff
@@ -94,3 +90,39 @@ export function useRelayNumberSuggestions(): SWRResponse<
 
   return relayNumberSuggestions;
 }
+
+export type RelayNumberSuggestion = {
+  friendly_name: string;
+  iso_country: string;
+  locality: string;
+  phone_number: string;
+  postal_code: string;
+  region: string;
+};
+
+/**
+ * Search folr relay number suggestions
+ */
+export const search = async (search: string) => {
+  // return early if search is empty
+  if (search.length === 0) return;
+
+  // if search is a number, assume it is an area code
+  // if search is not a number, assume it is a location
+  const searchParameter = !isNaN(+search)
+    ? `?area_code=${search}`
+    : `?location=${search}`;
+
+  // use api to search for relay number suggestions based on search body
+  const response = await apiFetch(`/relaynumber/search/${searchParameter}`, {
+    method: "GET",
+  });
+
+  if (!response.ok) {
+    throw new FetchError(response);
+  }
+
+  const data: RelayNumberSuggestion[] = await response.json();
```

Comment:
Oh neat, I didn't realise this API just returned data in the same format as the suggestions API - that makes things easier :)